reactNative/EthereumProject/TransactionsActions.js

```diff
@@ -0,0 +1,6 @@
+export const addTransactions = transactionsIndex => (
+    {
+      type: 'SELECT_TRANSACTION',
```

Comment:
don't use hard-coded strings. In this case you can either declare it as a const on the top of its file, or create a `constants.js` in the file structure I suggested above and put it there.